packages/accounts/src/nani-account/utils.ts

```diff
@@ -1,19 +1,19 @@
-import type { Address, Chain } from "viem";
 import {
   arbitrum,
   arbitrumGoerli,
   arbitrumSepolia,
   base,
   baseGoerli,
   baseSepolia,
-  sepolia,
   mainnet,
   optimism,
   optimismGoerli,
   optimismSepolia,
   polygon,
   polygonMumbai,
-} from "viem/chains";
+  sepolia,
+} from "@alchemy/aa-core";
```

Comment:
this is a stop gap solution. We will need to remove our dependency on this / make this better as a follow up. for now, we preserve the URLs in aa-core and rexport the chains we support.